External/FEXCore/Source/Interface/Core/JIT/Arm64/ALUOps.cpp

```diff
@@ -331,6 +346,40 @@ DEF_OP(Or) {
   }
 }
 
+DEF_OP(Orlshl) {
+  auto Op = IROp->C<IR::IROp_Orlshl>();
+  const uint8_t OpSize = IROp->Size;
+  const auto EmitSize = OpSize == 8 ? ARMEmitter::Size::i64Bit : ARMEmitter::Size::i32Bit;
+
+  const auto Dst = GetReg(Node);
+  const auto Src1 = GetReg(Op->Src1.ID());
+
+  uint64_t Const;
+  if (IsInlineConstant(Op->Src2, &Const)) {
```

Comment:
I don't believe this InlineConstant path in orlshl and orlshr will ever trigger, Since you added constprop support, but not inline constant support.
Probably want to look at ConstProp.cpp and the locations that use the `OP_OR` operation and then do `CreateInlineConstant` after it. Roughly on line 1049 in that file.

Reply:
So far I can't trigger any case that would inline constants either so meh.

Reply:
Yeah, I think for now this may be unreachable. OTOH I don't see a good reason to remove it because this could easily become useful later.